ui/desktop/app/components/hidden-secret/index.hbs

```diff
@@ -0,0 +1,11 @@
+<div class='hidden-secret'>
+  {{! TODO: The span wrap within a div because later we will add the toggle secret icon/button}}
+  <div>
+    <span>{{t 'resources.target.host-set.secret_label'}}</span>
```

Comment:
To skip the `no-bare-strings` rule I decided to add the secret characters as a translation string.